scripts/clean/db.sh

```diff
@@ -1,9 +1,15 @@
+#!/usr/bin/env bash
+set -e
+set -o pipefail
+
 # Cleans all data according to docs to prepare for wallets running:
 #     https://cardanodocs.com/technical/wallets/
 
 # TODO: support purescript API rebuilding?
 
-rm -rf ./run/*
+echo "Cleaning Cardano SL db..."
+
+rm -rf run/*
 rm -rf wallet-db/
 rm -rf db-abc/
```

Comment:
These are all `abc`s in scripts:
```
scripts/clean/db.sh:rm -rf db-abc/
scripts/clean/db.sh:rm -rf kademlia-abc.dump
scripts/launch/qa.sh:    --log-config scripts/log-templates/log-config-abc.yaml \
scripts/generate/genesis.sh:function abc {
scripts/generate/genesis.sh:abc 2>&1 | tee genesis.info
```

So `db-abc` appears only in clean script and hence it is useless. Same for `kademlia-abc.dump`.

`qa.sh` uses `db-qanet` and `kademlia-qanet.dump`. I think these names are reasonable. But cleaning script should be fixed. Also there is `log-config-abc.yaml` and this name it bad.